tools/make-wrapper-dlang

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
```

Comment:
This script could use some nice pretty printing for progress but better to do it if it actually proves itself used.